lib/src/components/NxDropdown/types.ts

```diff
@@ -36,6 +37,7 @@ export type Props = Omit<HTMLAttributes<HTMLDivElement>, 'className'> & {
   onCloseClick?: ((e: MouseEvent) => void) | null;
   toggleTooltip?: TooltipConfigProps | string | null;
   menuRef?: Ref<HTMLDivElement>;
+  menuId?: string;
```

Comment:
To be compatible with `propTypes` optional properties should allow both `undefined` and `null`

Reply:
> To be compatible with `propTypes` optional properties should allow both `undefined` and `null`

Fixed: d9fb143850b74df0bc73bc5fb767cc6b9f607bdd